energy:
   avg_speed: 1 # m/s
   charge_time: 2.4 # hours needed for charging
   moving_consumption: 30 # watt
   standing_consumption: 20 # watt
   battery_charge: 252 # Wh